tests/analysis_tests/tests-reanalyze/deadcode/expected/deadcode.txt

```diff
@@ -28,16 +28,6 @@
   addValueReference ComponentAsProp.res:12:24 --> ComponentAsProp.res:12:13
   addValueReference ComponentAsProp.res:13:16 --> React.res:3:0
   addValueReference ComponentAsProp.res:11:14 --> ComponentAsProp.res:6:34
-  addValueReference ComponentAsProp.res:9:6 --> ComponentAsProp.res:6:12
```

Comment:
A few more of these missing - perhaps another exp apply with loc?

Reply:
@cristianoc, I'm not entirely sure how this works, but since `addValueReference ComponentAsProp.res:9:6 --> ComponentAsProp.res:6:12` is mentioned a few lines above in this file, could this mean that it's fine and there's no risk of duplication? 

Or do you think we should aim for no changes in this file? Does it matter?